platformcomponents/desktop/button.json

```diff
@@ -60,6 +60,25 @@
         "background": "@theme-common-button-primary-disabled",
         "text": "@theme-common-text-disabled"
       }
+    },
+    "common-secondary": {
+      "#normal": {
+        "background": "@theme-common-button-secondary-normal",
+        "text": "@theme-common-text-white",
+        "border": "@theme-common-outline-button-normal"
+      },
+      "#hovered": {
+        "background": "@theme-common-button-secondary-hover",
+        "text": "@theme-common-text-white"
+      },
+      "#pressed": {
+        "background": "@theme-common-button-secondary-pressed",
```

Comment:
I see theme-common-buttons-secondary-active-normal on this one, not common-button-secondary-pressed
https://www.figma.com/file/vdL18BATeJAIq2JvGAjRPD/Components---Web-Application?node-id=4445%3A3906